### ISSUE TITLE
Feature request: Add titles to buttons/icons with no text

### ISSUE THREAD

**Issue by thekanbo** [fusioncms/fusioncms#411]
### Is your feature request related to a problem? Please describe.
Many icons and buttons in the control panel do not have text associated with them, and have no descriptive titles when hovered, meaning there is no way for the user to know what the control does.

Right now, many of these exist in the user/asset selection modals, or in the file manager interface.

### Describe the solution you'd like
These elements should have _at least_ a `title` attribute, so that the text displays automatically on hover.

### Feature request checklist
- [x] This is a single feature (i.e. not a re-write of the entire CMS)
- [x] The title starts with "Feature request: " and is followed by a clear feature name (Ex: `Feature request: add konami code to control panel`)
- [x] I have [searched for existing issues](https://github.com/fusioncms/fusioncms/issues) and to the best of my knowledge this is not a duplicate

**Comment by K-E-L** [comment]
closed by fusioncms/cms#370
